src/Model/Model.php

```diff
@@ -30,38 +30,43 @@ public function __construct (ManagerRegistry $registry)
     /**
      * @inheritDoc
      */
-    public function add (object $entity) : void
+    public function add (object $entity)
```

Comment:
For the very few projects that are already using `Model`, this is a breaking change. Do we simply ignore this fact as we have only upgraded a few projects to v6?

Reply:
Thought about that as well, haven’t decided yet. Probably a v7.